### DISCUSSION TITLE
Change billing date

### DISCUSSION THREAD

**Discussion by levilu800b** [community/community#110833] [New to GitHub]
I would like to change my billing dates. I would like that my Direct debit billing to be every 1st of each month rather than the 7th of every month.

**Comment by notSaqlain** [comment]
Unfortunately, **GitHub doesn't currently offer the option to directly change your billing date.** Their system automatically manages the billing date based on the initial setup and chosen billing cycle (monthly or yearly).

However, there are two things you can consider:

**1.** Change your billing cycle: You can switch your current billing cycle (monthly or yearly) to a new one that aligns better with your desired date. For example, if your current billing date is the 7th of each month and you prefer the 1st of each month, you can switch to a different monthly billing cycle that starts on the 1st.
Here's how to change your billing cycle on GitHub:

~Go to your Settings page.
~Click on Billing and plans in the "Access" section.
~Under Billing summary (for organizations) or Plans and usage (for personal accounts), find the option to Change plan.
~Select the desired plan with the billing cycle that aligns with your preferred date (e.g., starting on the 1st of each month).

**Note:** Changing your billing cycle might affect your current billing period and pricing, so be sure to review the details before confirming the switch.

**2.** Update your payment method: If you're open to changing your payment method, you could consider using a credit card instead of a direct debit. With a credit card, the payment date is usually fixed, so you wouldn't face any discrepancies with your preferred date. However, keep in mind the potential fees and limitations associated with using a credit card compared to a direct debit.

Remember, while you can't directly change your billing date within the same billing cycle, you can explore these alternative options to achieve a similar outcome and align your payments with your desired date.

[mark as complete if this helps, Thanks!]

**Comment by queenofcorgis** [comment]
Hi there @levilu800b - I suggest opening a ticket on our [Support page](https://support.github.com/contact). They can help assess if this is possible and discuss further details. Cheers!
